TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/console_link/console_link/models/metadata.py

```diff
@@ -13,47 +13,12 @@
 
 logger = logging.getLogger(__name__)
 
-FROM_SNAPSHOT_SCHEMA = {
-    "type": "dict",
-    # In the future, there should be a "from_snapshot" and a "from_live_cluster" option, but for now only snapshot is
-    # supported, so this is required. It _can_ be null, but that requires a snapshot to defined on its own in
-    # the services.yaml file.
-    "required": True,
-    "nullable": True,
-    "schema": {
-        "snapshot_name": {"type": "string", "required": True},
-        "otel_endpoint": {"type": "string", "required": False},
-        "local_dir": {"type": "string", "required": False},
-        "s3": {
-            'type': 'dict',
-            "required": False,
-            'schema': {
-                'repo_uri': {'type': 'string', 'required': True},
-                'aws_region': {'type': 'string', 'required': True},
-            }
-        },
-        "fs": {
-            'type': 'dict',
-            "required": False,
-            'schema': {
-                'repo_path': {'type': 'string', 'required': True},
-            }
-        }
-    },
-    # We _should_ have the check below, but I need to figure out how to combine it with a potentially
-    # nullable block (like this one)
-    # 'check_with': contains_one_of({'s3', 'fs'})
-
-}
-
 SCHEMA = {
-    "from_snapshot": FROM_SNAPSHOT_SCHEMA,
```

Comment:
I think we might need to reorganize the layout of the existing services.yaml, e.g. the snapshot definition seems like it should be separate from the snapshot tool